packages/flame_tiled/lib/src/renderable_tile_map.dart

```diff
@@ -212,11 +251,51 @@ abstract class _RenderableLayer<T extends Layer> {
 
   void refreshCache() {}
 
+  double get offsetX {
```

Comment:
shorter?

double get offsetX => layer.offsetX + (parent?.offsetX ?? 0);

Reply:
I prefer the more verbose version here, too much going on on one line of code for me. The long version groks instantly and the short version makes me scan very carefully for the double null coalesce 🤷

Honestly doesn't matter though

Reply:
sounds fine to me!

Reply:
I think the shorter version is actually easier to read, but up to you!

Reply:
You two have convinced me, although I'll put the getter definition on a new line inside a block